schema/time/index.js

```diff
@@ -0,0 +1,33 @@
+import { GraphQLString, GraphQLBoolean, GraphQLInt, GraphQLObjectType } from "graphql"
+
+export const timeFields = () => {
+  return {
+    day: { type: GraphQLInt },
+    wday: { type: GraphQLInt },
+    month: { type: GraphQLInt },
+    year: { type: GraphQLInt },
+    hour: { type: GraphQLInt },
+    min: { type: GraphQLInt },
+    sec: { type: GraphQLInt },
+    dst: { type: GraphQLBoolean },
+    unix: { type: GraphQLInt },
+    utc_offset: { type: GraphQLInt },
+    zone: { type: GraphQLString },
+    iso8601: { type: GraphQLString },
+  }
+}
+
+export const TimeType = new GraphQLObjectType({
+  name: "Time",
+  fields: () => ({
+    ...timeFields(),
+  }),
```

Comment:
Could shorten this to `fields: () => timeFields()` unless additional properties expect to be merged.

Reply:
Ah yeah, this was just left over from another part of the codebase I was referencing (which added the `cached` field).

Reply:
Or just even `fields: timeFields`, seeing as it is a function that returns an object of fields.